torch/_meta_registrations.py

```diff
@@ -7322,6 +7323,29 @@ def softmax(x: Tensor, dim: int, half_to_float: bool) -> Tensor:
     return res
 
 
+@register_meta(aten.constant_pad_nd)
+@out_wrapper()
+def _constant_pad_nd_meta(input, pad, value=0):
+    input_sizes = input.shape
```

Comment:
why avoiding moving.

   torch._check(
        len(pad) % 2 == 0,
        lambda: f"Length of pad must be even but instead it equals {len(pad)}",
    )?